assignments/objects.js

```diff
@@ -18,27 +18,69 @@ const example = {
 }
 
 // Write your intern objects here:
+const firstIntern = {
+  id: 1, 
+  email: "mmelloy0@PushSubscription.edu",
+  firstName: "Mitzi",
+  gender: "F"
+};
+
+const secondIntern = {
+  id: 2,
+  email: "kdiben1@tinypic.com",
+  firstName: "Kennan",
+  gender: "M",
+  speak: function(sayHello) {
+    console.log(`Hello, my name is ${this.firstName}!`);
+}
+};
+
+const thirdIntern = {
+  id: 3,
+  email:"kmummery2@wikimedia.org",
+  firstName: "Keven",
+  gender: "M"
+};
+
+const forthIntern = {
+  id: 4,
+  email: "gmartinson3@illinois.edu",
+  firstName: "Gannie",
+  gender: "M"
+};
+
+const fifthIntern = {
+  id: 5,
+  email: "adaine5@samsung.com",
+  firstName: "Antonietta",
+  gender: "F",
+  multiplyNums: function (num1, num2) {
+    console.log(num1 * num2);
+  }
+};
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(firstIntern.firstName);
 // Kennan's ID
-
+console.log(secondIntern.id);
 // Keven's email
-
+console.log(thirdIntern.email);
 // Gannie's name
-
+console.log(forthIntern.firstName);
 // Antonietta's Gender
-
+console.log(fifthIntern.gender);
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+console.log(secondIntern.speak())
```

Comment:
nice. You can also, without going back to the object, create a method in this line as well:

```
kennan.speak = function() {
  return "Hello, my name is Kennan!";
}
console.log(kennan.speak());
```